tslint.json

```diff
@@ -1,40 +1,38 @@
 {
-  "extends": "tslint:recommended",
-  "rules": {
-    "object-literal-sort-keys": false,
-    "no-console": [
-      false
+    "extends": [
+        "tslint:recommended",
+        "tslint-config-prettier",
+        "tslint-plugin-prettier"
     ],
-    "interface-name": [
-      true,
-      "never-prefix"
-    ],
-    "semicolon": [
-      true,
-      "never"
-    ],
-    "array-type": [
-      true,
-      "generic"
-    ],
-    "quotemark": [
```

Comment:
are these removed intentionally: 'quotemark', 'radix'

Reply:
Yeah, quotemark is now handled by prettier and radix I just fixed in the code.

Reply:
👍